src/App.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <v-app id="app">
-    <navigation-drawer />
+    <navigation-drawer/>
```

Comment:
We have generally kept the space before the self closing /
```suggestion
    <navigation-drawer />
```